worlds/alttp/__init__.py

```diff
@@ -388,6 +397,7 @@ def generate_output(self, output_directory: str):
                                world.menuspeed[player].current_key,
                                world.music[player],
                                world.sprite[player],
+                               world.oof[player],
```

Comment:
actually just had a generation issue since this attribute doesn't exist at run time. is this used by the adjuster or do we rip this out until implementing as a generation option?

Reply:
I don't think it does. I just removed it.